act-rules-format.bs

```diff
@@ -129,38 +155,34 @@ Language, either written or spoken, contained in nodes of the DOM [[DOM]] or acc
 
 <div class=example>
-  <p>A rule for labels of HTML `input` elements may have the following [expectations](#expectations):</p>
-  <ol>
-    <li>The test target has an accessible name (as described in [Accessible Name and Description: Computation and API Mappings 1.1](https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd_te)) [[accname-aam-1.1]].</li>
-    <li>The accessible name describes the purpose of the test target.</li>
-  </ol>
+  <p>The applicability of an atomic rule testing [WCAG 2.1 Audio Control](https://www.w3.org/TR/WCAG21/#audio-control):</p>
+  <blockquote>
+    <p>Any <code>video</code> or <code>audio</code> element(s) with the <code>autoplay</code> attribute, as well as any <code>object</code> element(s) that is used for automatically playing video or audio when the web page loads.</p>
+  </blockquote>
 </div>
 
 
-Expectations {#expectations}
-------------
+Expectations {#test-expectations}
+---------------------------------
 
-An ACT Rule MUST contain one or more expectations. An expectation is an assertion that must be true about each [test target](#output-test-target) (see [Applicability](#test-applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning isn't fully quantifiable.
+An [atomic rule](#atomic-rule-outline) MUST contain one or more expectations. An expectation is an assertion that must be true about each [test target](#output-test-target) (see [Applicability](#test-applicability)). Each expectation MUST be distinct, unambiguous, and be written in plain language. Unlike in applicability, a certain level of subjectivity is allowed in expectations. Meaning that the expectation has only one possible meaning, but that meaning isn't fully quantifiable.
 
-<p class=note>
-  Editor note - The task force is looking for feedback about whether expectations should be allowed to reference each other, or if each must be testable independently of the others.
-  For details, see [Github issue #173](https://github.com/w3c/wcag-act/issues/173).
-</p>
-
-When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations is false, the test target `failed` the rule. If the rule is part of a rule group, a test target that `failed` a rule may still meet the requirement (see [Rule Grouping](#grouping) for details).
+When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations is false, the test target `failed` the rule. If the atomic rule is used in a [composed rule](#composed-rule-outline), the composed rule may be `passed` when the atomic rule `failed`, depending on the [aggregation definition](#aggregation-definition) of the composed rule.
```

Comment:
"If one or more expectations is false" --> "If one or more expectations are false"